src/Parser/DeprecatedHelperVisitor.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace mglaman\PHPStanDrupal\Parser;
+
+use PhpParser\Node;
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitor;
+use PhpParser\NodeVisitorAbstract;
+
+class DeprecatedHelperVisitor extends NodeVisitorAbstract implements NodeVisitor
+{
+
+    private ?Node\Arg $deprecatedCall = null;
+
+    public function enterNode(Node $node)
+    {
+
+        if ($node instanceof Node\Expr\StaticCall) {
+            if ($node->class instanceof Node\Name\FullyQualified && $node->class->isFullyQualified() && $node->class->toString() === 'Drupal\Component\Utility\DeprecationHelper') {
```

Comment:
It's always an instance of FullyQualified even if imported, right? Pretty sure but double checking.

Reply:
yeah i removed this one after your review